addons/sourcemod/scripting/shop/item_manager.sp

```diff
@@ -1524,6 +1527,9 @@ bool ItemManager_FillItemsOfCategory(Menu menu, int client, int source_client, i
 		h_arCategories.GetString(category_id, category, sizeof(category));
 		do
 		{
+			if(shouldTest && result)
```

Comment:
Is it something, that can break jumptokey through keyvalues logic?
Is that tested properly?

Reply:
This will work only if there is a check for the presence of items in the category, this break is here so that the script does not check all items, but checks only one positive result and returns true if there was at least one successful check, because why we check all the items in the category if we can check one positive one and draw the category with at least it. Moreover, the category is checked here, and items are not added to it ...